### DISCUSSION TITLE
My student status has been renewed however I'm unable to access my benefits due to a wait period. I want to apply to become a Campus Expert but the deadline is only 2 days away, how to go about this?

### DISCUSSION THREAD

**Discussion by s11ngh** [community/community#65405] [GitHub Education]
### Hi everyone,

I had a Github Student Developer Pack for a year already. Few days ago I saw it had expired so after an approval for this year, I'm still unable to access my benefits. Without access to them I'm unable to access the application for Github Campus Expert program. I'm afraid I'll miss the deadline which is Sept 1, is there any way to get support for this?

**Comment by arafajs** [comment]
it usually takes 72 hours for the benefits to be reactivated after your application is approved. However, during busy periods like the start of a new semester, it may take a bit longer.
